src/main/java/com/textmessenger/model/entity/dto/Token.java

```diff
@@ -0,0 +1,13 @@
+package com.textmessenger.model.entity.dto;
+
+import lombok.Data;
+
+@Data
+public class Token {
+  private String accessToken;
+  private static final String tokenType = "Bearer";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this final field static too. ([squid:S1170](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1170))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=142&issues=AWWFkfgnX2sY87p0v-Gp&open=AWWFkfgnX2sY87p0v-Gp)

[ISSUE_KEY]: AWWFkfgnX2sY87p0v-Gp